### PR TITLE
[Concept Entry]: Added bindec in the terms folder of math-function

### LINKED ISSUE #2504: [Term Entry] PHP Math Functions bindec()
### Reason/inspiration (optional)

We would like a new term entry for [PHP](https://www.codecademy.com/resources/docs/php) about the `bindec()` function.

The entry should go in a new file in `docs/content/php/concepts/math-functions/terms/bindec/bindec.md`

The entry should 

- Have a syntax section illustrating the function with a ` ```pseudo ` code block. 
- An example showing the function in use with a ` ```php ` code block. 
- A runnable codebyte example in a ` ```codebyte/php` block. 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by dakshdeepHERE**
### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

Fixes #2504

### CODE REVIEW

File: content/php/concepts/math-functions/terms/bindec/bindec.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Methods'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The `bindec()` function in PHP is used to convert a binary string representation to its decimal equivalent. It takes a binary string as input and returns the corresponding decimal value.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/bindec/bindec.md:17]:
```suggestion
The **`bindec()`** function in PHP is used to convert a binary string representation to its decimal equivalent. It takes a binary string as input and returns the corresponding decimal value.
```

File: content/php/concepts/math-functions/terms/bindec/bindec.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+## Syntax
+
+Here's an explanation of the `bindec()` function along with syntax:
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/bindec/bindec.md:21]: We don't need this sentence. Let's remove it.

File: content/php/concepts/math-functions/terms/bindec/bindec.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+```pseudo
+bindec(string $binary_string): int
+```
+
+- **Parameters**:
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/bindec/bindec.md:27]: Let's remove the `**Parameters**` and `**Return Value**` and just have the two bullet points.

File: content/php/concepts/math-functions/terms/bindec/bindec.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+The output for the above code will be:
+
+```shell
+42
+```
+
+In the above example, the binary string `101010` is converted to its decimal equivalent, which is `42`.
+
+## Example 2
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/bindec/bindec.md:50]: We don't need a second example, let's remove it.

File: content/php/concepts/math-functions/terms/bindec/bindec.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+The output for the above code will be:
+
+```shell
+101
+```
+
+In this example, the binary string `1100101` is converted to its decimal equivalent, which is `101`.
+
+It's important to note that if the binary string contains characters other than `0` and `1`, or if it's an empty string, the `bindec()` function will return `0`. Additionally, if the binary string is too large to fit into an integer, the function may return an incorrect result due to integer overflow.
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/bindec/bindec.md:66]:
```suggestion
> **Note:** If the binary string contains characters other than `0` and `1`, or if it's an empty string, the `bindec()` function will return `0`. Additionally, if the binary string is too large to fit into an integer (defined by the system, usually 32 or 64 bits) the function may return an incorrect result due to integer overflow.
```

File: content/php/concepts/math-functions/terms/bindec/bindec.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+## Additional details
```
Comment (SSwiniarski) [inline content/php/concepts/math-functions/terms/bindec/bindec.md:68]: Most of these are redundant with the information you've given in the above note. We can remove this.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@dakshdeepHERE I did a first review and left a few comments.

**Comment by dakshdeepHERE** [comment]
@SSwiniarski all the changes are done can you check it once if there's anything else needed.

**Review by SSwiniarski — APPROVED** [review]
@dakshdeepHERE Looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@dakshdeepHERE @SSwiniarski LGTM

**Comment by dakshdeepHERE** [comment]
Thank you @caupolicandiaz

**Pull request merged**